src/Epic/index.stories.tsx

```diff
@@ -1,83 +1,116 @@
 import React, { ReactElement } from 'react';
-import { withKnobs, text } from '@storybook/addon-knobs';
 import { StorybookWrapper } from '../utils/StorybookWrapper';
 import { BrazeEndOfArticleComponent } from '../BrazeEndOfArticleComponent';
 import { knobsData } from '../utils/knobsData';
+import { coreArgTypes, ophanComponentIdArgType } from '../storybookCommon/argTypes';
+import { BrazeMessageProps } from '.';
+
+const paragraphDocs = Array.from({ length: 10 }, (_, idx) => {
```

Comment:
Could this be re-usable as a function that takes the number of paragraphs to add? This one uses 10, but there are quite a few that use 2.

I don't feel strongly! Abstraction has a cost, and there's no harm in being explicit if that's clear enough. For 2 paragraphs it's fine as-is, so up to you if it seems simpler with less duplication.

Reply:
Good question, I think for me the 2 paragraph cases don't quite warrant the abstraction and I'd rather leave them simple/explicit. Your suggestion has made me realise that I should assign the number of paragraphs to a `const` and use that, to make it clear what it is. (It should also be 9 not 10!)